config/values/30-istio-values.yml

```diff
@@ -0,0 +1,11 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
+---
+istio:
+  #! build_version is updated by build/istio/build.sh, do not edit it manually
+  build_version: "1.6.4"
+  upgrade_sidecars:
+    image: gcr.io/cf-networking-images/cf-k8s-networking/upgrade-sidecars@sha256:8cef1082ec0eb1d2e2f78c9b668065ce106de775cec929c745ca0e7f20a4c8fd
+
+  istiod:
+    replicas: 1
```

Comment:
Please see PR #427 for our proposed scaling interface.

Reply:
See comment here: https://github.com/cloudfoundry/cf-for-k8s/pull/429#discussion_r486690628